### DISCUSSION TITLE
%EB%8B%A4%EB%A5%B8-%EC%82%AC%EB%9E%8C%EC%97%90%EA%B2%8C-%EB%8F%84%EC%9B%80%EC%9D%84-%EA%B5%AC%ED%95%98%EB%8A%94-%EA%B0%80%EC%9E%A5-%EC%A2%8B%EC%9D%80-%EB%B0%A9%EB%B2%95/

### DISCUSSION THREAD

**Discussion by giscus** [heli-os/profile-giscus#4] [Comments]
# %EB%8B%A4%EB%A5%B8-%EC%82%AC%EB%9E%8C%EC%97%90%EA%B2%8C-%EB%8F%84%EC%9B%80%EC%9D%84-%EA%B5%AC%ED%95%98%EB%8A%94-%EA%B0%80%EC%9E%A5-%EC%A2%8B%EC%9D%80-%EB%B0%A9%EB%B2%95/

볼타 CTO 테오 블로그

https://dataportal.kr/%EB%8B%A4%EB%A5%B8-%EC%82%AC%EB%9E%8C%EC%97%90%EA%B2%8C-%EB%8F%84%EC%9B%80%EC%9D%84-%EA%B5%AC%ED%95%98%EB%8A%94-%EA%B0%80%EC%9E%A5-%EC%A2%8B%EC%9D%80-%EB%B0%A9%EB%B2%95/

**Comment by lleellee0** [comment]
개추드립니다

**Comment by dd55-na** [comment]
뻘글인데 이런 삽화 이미지는 어디서 구하세요? :D 
너무  적절하면서도 인상적이에요.

**Reply by heli-os** [reply]
ㅋㅋㅋㅋ AI로 만들어서 씁니다!! 그때그때 프롬프팅해서 뽑아내고 있습니다.

**Reply by dd55-na** [reply]
ㅋㅋㅋㅋㅋ 어쩐지 어디에도 본적이 없는 느낌이었습니다.

**Comment by gosekose** [comment]
도와주세요 ㅎㅎ !

**Comment by Invidam** [comment]
좋은 질문이 무엇일까 하고 고민해본 적이 있었는데 이렇게 체계적으로 적어주시니 감탄스럽네요..

어떻게하면 좋은 질문을 할 수 있을까?에 대해서도 생각해봤는데 중요한 건 많이 시도해보는 것 같아요.
조금은 부족하더라도, 원하는 답변을 못받더라도 질문을 할 용기가 있어야 좋은 질문을 할 수 있는 사람이 되는 것 같아요.
